### PR TITLE
Add new land use sample

### PR DESCRIPTION
**Pull request by jwood803**
Updating land use sample.

### CODE REVIEW

**Comment by jwood803** [comment]
@luisquintanilla @briacht The updated land use sample should now be ready for review

File: samples/modelbuilder/ImageClassification_Azure_LandUse/LandUse/LandUse.training.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        /// </summary>
+        /// <param name="mlContext"></param>
+        /// <returns></returns>
+        public static IEstimator<ITransformer> BuildPipeline(MLContext mlContext)
+        {
+            // Data process configuration with pipeline data transformations
+            var pipeline = mlContext.Transforms.LoadImages(outputColumnName:@"input1",imageFolder:@"",inputColumnName:@"ImageSource")      
+                                    .Append(mlContext.Transforms.ResizeImages(imageWidth:224,imageHeight:224,outputColumnName:@"input1",inputColumnName:@"input1",cropAnchor:ImageResizingEstimator.Anchor.Center,resizing:ImageResizingEstimator.ResizingKind.IsoCrop))      
+                                    .Append(mlContext.Transforms.ExtractPixels(outputColumnName:@"input1",inputColumnName:@"input1",colorsToExtract:ImagePixelExtractingEstimator.ColorBits.Rgb,orderOfExtraction:ImagePixelExtractingEstimator.ColorsOrder.ARGB))      
+                                    .Append(mlContext.Transforms.ApplyOnnxModel(modelFile:@"C:\Users\chron\Source\Repos\LandUse\LandUse\LandUse.onnx"));
```
Comment (luisquintanilla) [inline samples/modelbuilder/ImageClassification_Azure_LandUse/LandUse/LandUse.training.cs:33]: @jwood803 let's update this so it's a relative path.

File: samples/modelbuilder/ImageClassification_Azure_LandUse/LandUse_Console/LandUse.training.cs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        /// </summary>
+        /// <param name="mlContext"></param>
+        /// <returns></returns>
+        public static IEstimator<ITransformer> BuildPipeline(MLContext mlContext)
+        {
+            // Data process configuration with pipeline data transformations
+            var pipeline = mlContext.Transforms.LoadImages(outputColumnName:@"input1",imageFolder:@"",inputColumnName:@"ImageSource")      
+                                    .Append(mlContext.Transforms.ResizeImages(imageWidth:224,imageHeight:224,outputColumnName:@"input1",inputColumnName:@"input1",cropAnchor:ImageResizingEstimator.Anchor.Center,resizing:ImageResizingEstimator.ResizingKind.IsoCrop))      
+                                    .Append(mlContext.Transforms.ExtractPixels(outputColumnName:@"input1",inputColumnName:@"input1",colorsToExtract:ImagePixelExtractingEstimator.ColorBits.Rgb,orderOfExtraction:ImagePixelExtractingEstimator.ColorsOrder.ARGB))      
+                                    .Append(mlContext.Transforms.ApplyOnnxModel(modelFile:@"C:\Users\chron\Source\Repos\LandUse\LandUse\LandUse.onnx"));
```
Comment (luisquintanilla) [inline samples/modelbuilder/ImageClassification_Azure_LandUse/LandUse_Console/LandUse.training.cs:33]: @jwood803 let's update this so it's a relative path.

File: samples/modelbuilder/ImageClassification_Azure_LandUse/LandUse_WebApi/LandUse.training.cs [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        /// </summary>
+        /// <param name="mlContext"></param>
+        /// <returns></returns>
+        public static IEstimator<ITransformer> BuildPipeline(MLContext mlContext)
+        {
+            // Data process configuration with pipeline data transformations
+            var pipeline = mlContext.Transforms.LoadImages(outputColumnName:@"input1",imageFolder:@"",inputColumnName:@"ImageSource")      
+                                    .Append(mlContext.Transforms.ResizeImages(imageWidth:224,imageHeight:224,outputColumnName:@"input1",inputColumnName:@"input1",cropAnchor:ImageResizingEstimator.Anchor.Center,resizing:ImageResizingEstimator.ResizingKind.IsoCrop))      
+                                    .Append(mlContext.Transforms.ExtractPixels(outputColumnName:@"input1",inputColumnName:@"input1",colorsToExtract:ImagePixelExtractingEstimator.ColorBits.Rgb,orderOfExtraction:ImagePixelExtractingEstimator.ColorsOrder.ARGB))      
+                                    .Append(mlContext.Transforms.ApplyOnnxModel(modelFile:@"C:\Users\chron\Source\Repos\LandUse\LandUse\LandUse.onnx"));
```
Comment (luisquintanilla) [inline samples/modelbuilder/ImageClassification_Azure_LandUse/LandUse_WebApi/LandUse.training.cs:33]: Let's update this so it's a relative path.

File: samples/modelbuilder/ImageClassification_Azure_LandUse/README.md [hunk 4]
```diff
@@ -66,119 +65,10 @@ The original RGB version of the dataset which contains 27,000 images is also ava
 
 The goal of the application is to categorize what a plot of land is used for using satellite images. The Machine Learning task to use in this scenario is image classification. The model in this application was trained using Model Builder.
 
-[Model Builder](https://marketplace.visualstudio.com/items?itemName=MLNET.07) is an intuitive graphical Visual Studio extension to build, train, and deploy custom machine learning models.
+[Model Builder](https://docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/install-model-builder) is an intuitive graphical Visual Studio extension to build, train, and deploy custom machine learning models.
```
Comment (luisquintanilla) [inline samples/modelbuilder/ImageClassification_Azure_LandUse/README.md:68]: Removing localization 

```suggestion
[Model Builder](https://docs.microsoft.com/dotnet/machine-learning/how-to-guides/install-model-builder) is an intuitive graphical Visual Studio extension to build, train, and deploy custom machine learning models.
```

**Review by luisquintanilla — COMMENTED** [review]
@jwood803 looks great. Let's update some of the paths so they're relative and test that it works with those changes. Other than that we should be good to go. Thanks!

**Review by luisquintanilla — APPROVED** [review]

**Pull request merged**